systemtest/src/test/resources/oauth2/create_realm.sh

```diff
@@ -0,0 +1,351 @@
+#!/bin/bash
+
+USERNAME=$1
+PASSWORD=$2
+URL=$3
+
+TOKEN=$(curl --insecure -X POST -d "client_id=admin-cli&client_secret=aGVsbG8td29ybGQtcHJvZHVjZXItc2VjcmV0&grant_type=password&username=$USERNAME&password=$PASSWORD" "$URL/realms/master/protocol/openid-connect/token" | awk -F '\"' '{print $4}')
+
+curl -v --insecure https://sso-oauth2-cluster-test.apps.amq-streams-july16.openshift-aws.rhocf-dev.com/auth/admin/realms \
+  -H "Authorization: Bearer $TOKEN" \
+  -H 'Content-Type: application/json' \
+  -H 'Postman-Token: 3a6cd746-03b5-46fe-a54a-014fc7c51983' \
```

Comment:
Do we need this over `curl`?

Reply:
I tried this import with multiple things as follows. `Keycloak Java API` and also `kcadm.sh` - which didn't work as expected